### PR TITLE
Fix resize

### PR DESCRIPTION
**Pull request by briankassouf**

### CODE REVIEW

File: server.go [hunk 1]
```diff
@@ -49,12 +49,15 @@ func (s *Server) _uploadFile(uploadFile io.ReadCloser, w http.ResponseWriter, fi
 	upload := uploadedfile.NewUploadedFile(fileName, tmpFile.Name(), "image/jpeg")
 	processor, err := imageprocessor.Factory(s.Config.MaxFileSize, upload)
 	if err != nil {
+		fmt.Println(err)
```
Comment (Jasdev) [inline server.go]: Do we wanna keep these logs?

File: server.go [hunk 2]
```diff
@@ -49,12 +49,15 @@ func (s *Server) _uploadFile(uploadFile io.ReadCloser, w http.ResponseWriter, fi
...
+
 		ErrorResponse(w, "Unable to process image!", http.StatusInternalServerError)
 		return
 	}
 
 	err = processor.Run(upload)
 	if err != nil {
+		fmt.Println(err)
```
Comment (Jasdev) [inline server.go]: Same as ^

File: imageprocessor/gm/gm.go [hunk 3]
```diff
@@ -34,12 +34,14 @@ func Quality(filename string, quality int) (string, error) {
...
 		"-quality",
-		string(quality),
+		fmt.Sprintf("%d", quality),
 		"-density",
 		"72x72",
-		outfile,
+		"JPEG:"+outfile,
 	}
 
+	fmt.Printf("%s -quality %d -density 72x72 %s", filename, quality, outfile)
```
Comment (Jasdev) [inline imageprocessor/gm/gm.go]: Need to keep logs?

File: imageprocessor/gm/gm.go [hunk 4]
```diff
@@ -54,10 +56,12 @@ func ResizePercent(filename string, percent int) (string, error) {
 	args := []string{
 		filename,
 		"-resize",
-		fmt.Sprintf("%i%", percent),
+		fmt.Sprintf("%d%%", percent),
 		outfile,
 	}
 
+	fmt.Printf("%s -resize %d%% -density 72x72 %s \n", filename, percent, outfile)
```
Comment (Jasdev) [inline imageprocessor/gm/gm.go]: same ^

File: imageprocessor/gm/gm.go [hunk 5]
```diff
@@ -34,12 +34,14 @@ func Quality(filename string, quality int) (string, error) {
...
 		filename,
 		"-quality",
-		string(quality),
+		fmt.Sprintf("%d", quality),
 		"-density",
 		"72x72",
 		outfile,
 	}
 
+	fmt.Printf("%s -quality %d -density 72x72 %s", filename, quality, outfile)
```
Comment (Jasdev) [inline imageprocessor/gm/gm.go]: Need to keep logs?

Comment (briankassouf) [inline imageprocessor/gm/gm.go]: I want to keep these ones for a bit

File: server.go [hunk 6]
```diff
@@ -46,7 +45,7 @@ func (s *Server) _uploadFile(uploadFile io.ReadCloser, w http.ResponseWriter, fi
 		return
 	}
 
-	upload := uploadedfile.NewUploadedFile(fileName, tmpFile.Name(), "image/jpeg")
+	upload := uploadedfile.NewUploadedFile("testfile.jpg", tmpFile.Name(), "image/jpeg")
```
Comment (Jasdev) [inline server.go]: accidentally overwrote merge

File: server.go [hunk 7]
```diff
@@ -80,7 +79,7 @@ func (s *Server) initServer() {
 	fileHandler := func(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Content-Type", "application/json")
 
-		uploadFile, header, err := r.FormFile("image")
```
Comment (Jasdev) [inline server.go]: overwrote merge

File: server.go [hunk 8]
```diff
@@ -88,7 +87,7 @@ func (s *Server) initServer() {
 			return
 		}
 
-		s._uploadFile(uploadFile, w, header.Filename)
```
Comment (Jasdev) [inline server.go]: same

File: server.go [hunk 9]
```diff
@@ -99,7 +98,7 @@ func (s *Server) initServer() {
 			return
 		}
 
-		s._uploadFile(uploadFile, w, "")
```
Comment (Jasdev) [inline server.go]: same

File: server.go [hunk 10]
```diff
@@ -26,7 +25,7 @@ func CreateServer(c *Configuration) *Server {
 	return &Server{c, httpclient, store}
 }
 
-func (s *Server) _uploadFile(uploadFile io.ReadCloser, w http.ResponseWriter, fileName string) {
```
Comment (Jasdev) [inline server.go]: take master

File: imageprocessor/gm/gm.go (hunk 5 above)
Comment (Jasdev) [inline imageprocessor/gm/gm.go]: :+1:

**Comment by briankassouf** [comment]
@Jasdev  fixed the merge conflict

File: imageprocessor/gm/gm.go [hunk 11]
```diff
@@ -54,10 +56,12 @@ func ResizePercent(filename string, percent int) (string, error) {
 	args := []string{
 		filename,
 		"-resize",
-		fmt.Sprintf("%i%", percent),
+		fmt.Sprintf("%d%%", percent),
```
Comment (lospro7) [inline imageprocessor/gm/gm.go]: escaping?

Comment (briankassouf) [inline imageprocessor/gm/gm.go]: Yeah two "%" escapes to one

**Pull request merged**